### ISSUE TITLE
Enable the compilation of the YarpRobotLogger device

### ISSUE THREAD

**Issue by GiulioRomualdi** [conda-forge/bipedal-locomotion-framework-feedstock#5]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

It would be nice to enable the compilation of the device

### Installed packages

```shell
None
```

### Environment info

```shell
None
```

**Comment by GiulioRomualdi** [comment]
cc @traversaro

**Comment by traversaro** [comment]
It is not already enable? I tought it was automatically compiled if YARP was found, see also https://github.com/robotology/robotology-superbuild/blob/ed17cdb1ff3804baba0e321b04da8411b41919ab/cmake/Buildbipedal-locomotion-framework.cmake where I can't find any explicit option to enable the logger.

**Comment by traversaro** [comment]
Indeed, in the package build log there is:
~~~
2022-11-11T13:05:03.9929440Z --  [ ] Plugin: FloatingBaseEstimatorDevice (ENABLE_FloatingBaseEstimatorDevice)
2022-11-11T13:05:03.9964510Z --  [ ] Plugin: YarpRobotLoggerDevice (ENABLE_YarpRobotLoggerDevice)
2022-11-11T13:05:04.0004619Z --  [ ] Plugin: VectorsCollectionWrapper (ENABLE_VectorsCollectionWrapper)
~~~

I just checked, and apparently we have the same problem in the superbuild:
~~~
2022-11-24T03:16:35.1038135Z --  [ ] Plugin: FloatingBaseEstimatorDevice (ENABLE_FloatingBaseEstimatorDevice)
2022-11-24T03:16:35.1038625Z --  [ ] Plugin: YarpRobotLoggerDevice (ENABLE_YarpRobotLoggerDevice)
2022-11-24T03:16:35.1039121Z --  [ ] Plugin: VectorsCollectionWrapper (ENABLE_VectorsCollectionWrapper)
~~~

**Comment by GiulioRomualdi** [comment]
This should be solved by #10

**Comment by traversaro** [comment]
Yes.
